recipes/boost_histogram/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "boost-histogram" %}
```

Comment:
Should this be called `boost-histogram` or `boost_histogram`?

Reply:
I think boost-histogram for consistency, unless there’s a reason to do  otherwise

Reply:
Sounds good, PyPI treats `_` and `-` as if they're the same in many places so for pip it doesn't matter but conda it does.